lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/dataaccess/DBODataAccessSubmissionDAOImpl.java

```diff
@@ -46,7 +54,27 @@
 			+ " AND "+COL_DATA_ACCESS_SUBMISSION_ACCESS_REQUIREMENT_ID+" = ?"
 			+ " AND ("+TABLE_DATA_ACCESS_SUBMISSION+"."+COL_DATA_ACCESS_SUBMISSION_CREATED_BY+" = ?"
```

Comment:
can we force submitter to be accessor and remove the left side of the OR statement